setup.py

```diff
@@ -8,6 +8,24 @@
 README_FILE = os.path.join(PROJECT_ROOT, "README.md")
 VERSION_FILE = os.path.join(PROJECT_ROOT, "bambi", "version.py")
 REQUIREMENTS_FILE = os.path.join(PROJECT_ROOT, "requirements.txt")
+REQUIRED_PYTHON_VERSION = (3, 7, 2)
+
+
+def check_installation(rv):
+    current_version = sys.version_info
+    if (
+        (current_version[0] == rv[0])
+        and (current_version[1] == rv[1])
+        and (current_version[2] < rv[2])
+    ):
+        sys.stderr.write(
+            f"[{sys.argv[0]}] - Error: Your Python interpreter must be "
+            + f"{rv[0], rv[1], rv[0]}or greater\n"
+        )
+        sys.exit(-1)
+    else:
+        pass
+    return 0
```

Comment:
is this a recommended practice? Otherwise we can remove it

Reply:
I don't think it's strictly necessary and can probably be removed